CentralUnit/Communication/XbeeNetwork.cs

```diff
@@ -0,0 +1,416 @@
+﻿// <copyright file="XbeeNetwork.cs" company="Moto Gymkhana">
+//     Copyright (c) Moto Gymkhana. All rights reserved.
+// </copyright>
+namespace Communication
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading;
+    using log4net;
+    using XBeeLibrary.Core.Models;
+    using XBeeLibrary.Core.Packet;
+    using XBeeLibrary.Core.Packet.Common;
+
+    /// <summary>
+    /// A network of <c>Xbee</c> devices accessed through a single serial communication channel.
+    /// </summary>
+    public class XbeeNetwork
```

Comment:
I think this class needs a stop and/or Dispose method for graveful shutdown

Reply:
Implemented IDisposable.